### ISSUE TITLE
Design GUI for media play area on starting page

### ISSUE THREAD

**Issue by i315315** [amosproj/amos2022ws02-automotive-test-app#42]
## User story
1. As a User
2. I want to be able to enter an area in the test app where all use cases for the media play are collected 
3. So that I can easily switch between the use cases.

With this User story the graphical user interface of the media play area of the app is to be designed.
The implementation of entering the area will be done with user story #43.

## Acceptance criteria
* The graphical user interface of the media play area in the app is designed.
* An XML for the GUI of the media play area in the app has been created in Android studio.

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by Andy3189** [comment]
Deleting Branches via pr accidently closes issue
